etc/bootstrap-example2/dn-node-1.toml

```diff
@@ -0,0 +1,41 @@
+# service node type, [DN|CN|LOG]
+service-type = "DN"
+
+[log]
+level = "debug"
+format = "json"
+max-size = 512
+
+[[fileservice]]
+# local fileservice instance, used to store TAE Data and DNStore metadata.
+name = "local"
+# use disk as fileservice backend
+backend = "DISK"
+# set the directory used by DISK backend. There must has a file named "thisisalocalfileservicedir"
+# in the data dir
+data-dir = "data-dir"
+
+[[fileservice]]
+# s3 fileservice instance, used to store data.
+name = "s3"
+# use disk as fileservice backend.
+backend = "DISK"
+# set the directory used by DISK backend. There must has a file named "thisisalocalfileservicedir"
+# in the data dir
+data-dir = "data-dir"
+
+[dn]
+uuid = "dn4dccb4-4d3c-41f8-b482-5251dc7a41bf"
```

Comment:
Please check whether this is a valid uuid. I think you can only have 0-9 and a-f chars.

Reply:
Yes, I can start a DN with this uuid.